banzai_floyds/binning.py

```diff
@@ -0,0 +1,9 @@
+from banzai.stages import Stage
```

Comment:
I broke the data binning into its own stage to make testing easier and more modular.